scripts/build.js

```diff
@@ -41,6 +42,13 @@ const args = parseArgs(process.argv.slice(2), {
         litCssPlugin({
           uglify: true,
           filter: /components\/.*\.css$/,
+          // Match line containing ':hover'
+          transform: (content) => isProduction ? content : content.replace(/.*:hover[^{]*/g, matched => {
```

Comment:
I would not do that check in every cycle. Instead something like:

```js
const cssPluginOptions = {
  uglify: true,
  filter: /components\/.*\.css$/
};

if (isStorybook) {
   cssOptions.transform = (content) => content.replace( .... 
}

...

litCssPlugin(cssPluginOptions),
```

Reply:
Good catch. 👍